tests/components/lacrosse_view/test_sensor.py

```diff
@@ -71,3 +75,31 @@ async def test_field_not_supported(hass: HomeAssistant, caplog) -> None:
     assert entries[0].state == ConfigEntryState.LOADED
     assert hass.states.get("sensor.test_some_unsupported_field") is None
     assert "Unsupported sensor field" in caplog.text
+
+
+async def test_field_types(hass: HomeAssistant) -> None:
```

Comment:
Should this rather be parametrized (using the return values below) to check each sensor type separately to verify that the expected type is returned?